apps/openmw/mwmechanics/actors.cpp

```diff
@@ -1722,7 +1727,10 @@ namespace MWMechanics
             {
                 const float dist = (playerPos - iter->first.getRefData().getPosition().asVec3()).length();
                 bool isPlayer = iter->first == player;
-                bool inRange = isPlayer || dist <= mActorsProcessingRange;
+                CreatureStats &stats = iter->first.getClass().getCreatureStats(iter->first);
+                int packageId = stats.getAiSequence().getTypeId();
+                bool travelling = (packageId == AiPackage::TypeIdTravel) || (packageId == AiPackage::TypeIdInternalTravel);
```

Comment:
I'd prefer to have a more generic "dependent on distance" flag in AI packages themselves.